content/docs/nextjs/github-public-repo.md

```diff
@@ -0,0 +1,291 @@
+---
+title: Open Authoring with GitHub
+id: /docs/nextjs/github-public-repo
+prev: /docs/nextjs/markdown
+next:
+---
+
+Using Next.js's preview-mode ([announced in Next.js 9.3](https://nextjs.org/blog/next-9-3)), we can load a separate set of data depending on if we are in "edit-mode" or "production-mode". This pairs well with the GitHub API, where we can provide an "Open Authoring" experience where anyone can fork your site, make changes, and create a pull request, all through the TinaCMS UI!
+
+## Requirements:
+
+- You must be using **Next.js v.9.3** or later.
+- If your site is static, is must be hosted on a service that supports Next.js preview-mode (E.g [Zeit's Now](https://zeit.co/home))
+- Your editable content must be stored in GitHub
+- Your repository containing your content **must be public**
+
+## Installation
+
+We'll need to add a few Tina-GitHub packages to our site:
```

Comment:
Do we note the required peerDeps? Assuming this is a fresh install and they may not have `tinacms`, `styled-components` etc.

Reply:
I was thinking the same thing. It would be good if this guide assumed you had nothing setup at all.